### PR TITLE
documentation for tilde processed archive

### PR DESCRIPTION
**Pull request by elidana**
Hello @rumachan and @CallumNZ ,

this pull request is adding information to our dataset tutorials to describe the recently added tilde time series dataset in the GeoNet AWS Open Data bucket.

Can you please review this, happy to change or add anything you can think about!

Thanks heaps!

### CODE REVIEW

File: AWS_Open_Data/README.md [hunk 1]
```diff
@@ -272,3 +273,86 @@ Data are organized by equipment type and date and objects' prefixes are formed a
...
+
+Additional information around this dataset can be found on the [GeoNet website](https://www.geonet.org.nz/).
+
+All time series data are available under the bucket and prefix `s3://geonet-open-data/time-series/`.
+
+### <a name="tilde"></a>Tilde Time Series data
+Time series data derived from GeoNet sensor network, repackaged and formatted via the GeoNet Tilde application.
+
+Details on Tilde data name format can be found on the [GeoNet Tilde API website page](https://tilde.geonet.org.nz/).
+Details on available dataset can be found on the [GeoNet Tilde Data Discovery website page](https://tilde.geonet.org.nz/ui/data-exploration).
```
Comment (CallumNZ) [inline AWS_Open_Data/README.md:294]: *datasets

File: AWS_Open_Data/README.md [hunk 2]
```diff
@@ -272,3 +273,86 @@ Data are organized by equipment type and date and objects' prefixes are formed a
...
+
+This dataset is the same that can be obtained from the Tilde API _data_ endpoint. For large data requests and bulk downloads, users are recommended to download data from this AWS Open Data bucket.
```
Comment (CallumNZ) [inline AWS_Open_Data/README.md:296]: *is the same one that..

File: AWS_Open_Data/README.md [hunk 3]
```diff
@@ -272,3 +273,86 @@ Data are organized by equipment type and date and objects' prefixes are formed a
...
+
+Data are provided in gzip compressed csv (Comma Separated Values) format.
+
+All time series data are available under the bucket and prefix `s3://geonet-open-data/time-series/tilde/`.
+
+prexix | content
```
Comment (CallumNZ) [inline AWS_Open_Data/README.md:302]: *prefix

File: AWS_Open_Data/README.md [hunk 4]
```diff
@@ -272,3 +273,86 @@ Data are organized by equipment type and date and objects' prefixes are formed a
...
+-- | --
+`time-series/tilde/` | Tilde time series data
+
+Tilde application has only been released in 2021 and we expect that the format of this dataset might change in the future. To allow our users to migrate their downstream procedures, object prefixes will include versioning.
```
Comment (CallumNZ) [inline AWS_Open_Data/README.md:306]: *The Tilde application

File: AWS_Open_Data/README.md [hunk 5]
```diff
@@ -272,3 +273,86 @@ Data are organized by equipment type and date and objects' prefixes are formed a
...
+
+prexix | content
+-- | --
+`time-series/tilde/` | Tilde time series data
+
+Tilde application has only been released in 2021 and we expect that the format of this dataset might change in the future. To allow our users to migrate their downstream procedures, object prefixes will include versioning. 
+
+For this dataset, object prefixes are following an _hive style_ partitioning. 
+
+Data are organized by version, domain, station, time series name, station sensor code, time series parameters (method and aspect) and time.
```
Comment (CallumNZ) [inline AWS_Open_Data/README.md:310]: I think it should just say `sensor code`, not `station sensor code`.

File: AWS_Open_Data/README.md [hunk 6]
```diff
@@ -272,3 +273,86 @@ Data are organized by equipment type and date and objects' prefixes are formed a
...
+For example, the `water-height` and `soil-moisture` time series, will only have these combinations available:
+
+name | domain | method | aspect | unit
+-----|--------|--------|--------|-----
+water-height | coastal/dart | 15m/15s/raw | nil | m
+soil-moisture | envirosensor | snapshot | 10-cm/20-cm/30-cm/40-cm/50-cm/60-cm/70-cm/80-cm | percent
+
+We are increasing the number of derived time series distributed via Tilde, so the list below will grow in time. 
+
+We recommend to refer to the [Tilde Data Discovery page](https://tilde.geonet.org.nz/ui/data-exploration) for an uptodate and quick overview of available datasets and possible prefix combinations.
```
Comment (CallumNZ) [inline AWS_Open_Data/README.md:358]: *up-to-date

**Review by CallumNZ — CHANGES_REQUESTED** [review]

File: AWS_Open_Data/README.md (hunk 1 above)
Comment (elidana) [reply AWS_Open_Data/README.md:294]: done

File: AWS_Open_Data/README.md (hunk 2 above)
Comment (elidana) [reply AWS_Open_Data/README.md:296]: done

File: AWS_Open_Data/README.md (hunk 3 above)
Comment (elidana) [reply AWS_Open_Data/README.md:302]: :facepalm: ... done

File: AWS_Open_Data/README.md (hunk 4 above)
Comment (elidana) [reply AWS_Open_Data/README.md:306]: fair enough! modified as you suggest

File: AWS_Open_Data/README.md (hunk 5 above)
Comment (elidana) [reply AWS_Open_Data/README.md:310]: done here and above

File: AWS_Open_Data/README.md (hunk 6 above)
Comment (elidana) [reply AWS_Open_Data/README.md:358]: done

**Comment by elidana** [comment]
thanks a lot @CallumNZ for your review! All suggested changes now added :)

**Review by CallumNZ — APPROVED** [review]

**Comment by elidana** [comment]
oh thanks a lot @rumachan for reviewing it!!!

**Pull request merged**